recipes/des-despyfits/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "des-despyfits" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DarkEnergySurvey/despyfits/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 0d96bbb440763e655c0ec35eb3961811f9e4c4bf06acb3863d830a1fe2176bbb
+  patches:
+    - libload.patch
+    - libload_again.patch
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

```suggestion
    - {{ compiler('c') }}
    - m2-git   # [win]
```

try adding m2-git if patching is failing.

---

ahhhhhh cool - I'll try that next! thank you!

---

It turns out the madness of this PR is that one patch needs CRLF line endings and the other does not. 🤦 

Thanks for the note. We cannot build on windows so resolving this one.

---

i sometimes just copy my patched file over.